### PR TITLE
Tracing middleware for APIGatewayHttpV2ApiProxyRequest

### PR DESCRIPTION
**Pull request by josep-bocanegra**
A tracing middleware for APIGatewayHttpV2ApiProxyRequest has been added along with its documentation and a sample.

### CODE REVIEW

**Review by fermins — APPROVED** [review]
Great!

File: samples/ApiGatewayHttpApiV2Tracing/Voxel.MiddyNet.ApiGatewayHttpApiV2TracingSample/serverless.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+package:
+  individually: true
+
+functions:
+  apiGatewayTracingSample:
+    handler: Voxel.MiddyNet.ApiGatewayHttpApiV2TracingSample::Voxel.MiddyNet.ApiGatewayHttpApiV2TracingSample.ApiGatewayHttpApiV2Tracing::Handler
+    package:
+      artifact: bin/Release/netcoreapp3.1/Voxel.MiddyNet.ApiGatewayHttpApiV2TracingSample.zip
+    events:
+      - http:
```
Comment (vgaltes) [inline samples/ApiGatewayHttpApiV2Tracing/Voxel.MiddyNet.ApiGatewayHttpApiV2TracingSample/serverless.yml:22]: Have we tried this? The httpApi event is httpApi, not http.

**Review by vgaltes — CHANGES_REQUESTED** [review]
I think the serverless.yml is incorrect.

File: samples/ApiGatewayHttpApiV2Tracing/Voxel.MiddyNet.ApiGatewayHttpApiV2TracingSample/serverless.yml (hunk 1 above)
Comment (fermins) [reply samples/ApiGatewayHttpApiV2Tracing/Voxel.MiddyNet.ApiGatewayHttpApiV2TracingSample/serverless.yml:22]: I'll try it, but I think @josep-bocanegra is right. It's the same construct we used for ApiGatewayTracingSample.

Comment (vgaltes) [reply samples/ApiGatewayHttpApiV2Tracing/Voxel.MiddyNet.ApiGatewayHttpApiV2TracingSample/serverless.yml:22]: I mean, if we´re doing this for the "new" HTTP Api, the event should be httpApi, as explained here: https://www.serverless.com/framework/docs/providers/aws/events/http-api/

Comment (josep-bocanegra) [reply samples/ApiGatewayHttpApiV2Tracing/Voxel.MiddyNet.ApiGatewayHttpApiV2TracingSample/serverless.yml]: Fixed!

**Review by vgaltes — APPROVED** [review]
Good job!

**Pull request merged**